### DISCUSSION TITLE
beginners python

### DISCUSSION THREAD

**Discussion by narmink** [snu-quiqcl/qiwis#212] [Q&A] [answered]
name = "What is up python?"

print(name)

nothing prints to my screen.  Am i doing something wrong? help!

**Comment by quinterobuong** [comment]
Im also a beginner,but hope this helps.
I've written the code just the way u have done, then
on the terminal, i typed "python name.py" and it perfectly worked.

**Comment by narmink** [comment, marked as answer]
Thank you for your feedback. I’ll try again. Sent from my iPhoneOn Aug 18, 2023, at 12:23 AM, quinterobuong ***@***.***> wrote:﻿
Im also a beginner,but hope this helps.
I've written the code just the way u have done, then
on the terminal, i typed "python name.py" and it perfectly worked.
